frontend/components/Editor.js

```diff
@@ -1046,6 +1058,27 @@ patch: ${JSON.stringify(
             }
         }
 
+        this.desktop_submit_file_change = async () => {
+            window.plutoDesktop?.ipcRenderer.once("PLUTO-MOVE-NOTEBOOK", async (/** @type {string | undefined} */ loc) => {
+                try {
+                    if (!!loc)
+                        await update_notebook((notebook) => {
+                            notebook.in_temp_dir = false
+                            notebook.path = loc
+                        }, false)
+                    // @ts-ignore
+                    document.activeElement?.blur()
```

Comment:
```suggestion
                    document.activeElement?.blur?.()
```

then you don't need the `@ts-ignore`